drivers/console/Kconfig

```diff
@@ -358,4 +358,10 @@ config EFI_CONSOLE
 	help
 	  Enable this option to use EFI console output.
 
+config WINSTREAM_CONSOLE
+	bool "Use Winstream console"
+	depends on WINSTREAM
+	help
+	  Use winstream as a console.
```

Comment:
If you're moving this out of the soc/ folder and into drivers/console, I think it is worth it to improve the help text here. I can't figure out what winstream is by googling the name. Is it something generic? SoC specific? Etc

Reply:
+1

I think the best description can for now be found here: https://github.com/zephyrproject-rtos/zephyr/blame/53ef68d4/include/zephyr/sys/winstream.h
```
 ** @brief Lockless shared memory byte stream IPC
 *
 * The sys_winstream utility implements a unidirectional byte stream
 * with simple read/write semantics on top of a memory region shared
 * by the writer and reader. ...
```

Maybe @andyross can provide the Kconfig? It's his baby :-)

Reply:
it depends on WINSTREAM which is well defined in its own entry, no need for googling :)
```
config WINSTREAM
        bool "Lockless shared memory window byte stream"
        help
          Winstream is a byte stream IPC for use in shared memory
          "windows", generally for transmit to non-Zephyr contexts that
          can't share Zephyr APIs or data structures.
```

will update.

Reply:
Ah. That's what I was missing. Thanks.